source/Ship.h

```diff
@@ -456,6 +461,14 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	const StellarObject *GetTargetStellar() const;
 	// Get ship's target system (it should always be one jump / wormhole pass away).
 	const System *GetTargetSystem() const;
+	// Targets for persistent ships (i.e. mission NPCs).
+	const bool HasTravelDirective() const;
```

Comment:
Primitive return types don't need to be `const`.
```suggestion
	bool HasTravelDirective() const;
```